src/azure_pdf_parser/convert.py

```diff
@@ -172,9 +184,16 @@ def azure_api_response_to_parser_output(
         PDFPageMetadata(
             # Azure page numbers start from an index of 1, at cpr our data starts from 0
             page_number=page.page_number - 1,
-            dimensions=(page.width, page.height),
+            # Azure units are in inches but our corpus is in 72ppi pixels
+            dimensions=(
+                page.width * DIMENSION_CONVERSION_FACTOR,
+                page.height * DIMENSION_CONVERSION_FACTOR,
+            ),
         )
         for page in api_response.pages
+        if page.width is not None
```

Comment:
`None not in (page.width, ...)` wasn't getting passed the pre-commit config for some reason.